target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -234,6 +234,21 @@ define Device/A4001N1
 endef
 TARGET_DEVICES += A4001N1
 
+define Device/AGPWI
+  $(Device/bcm63xx)
+  DEVICE_VENDOR := Telecom Italia
+  DEVICE_MODEL := ADSL2+ Wi-Fi N AGPWI
+  DEVICE_ALT0_VENDOR := ADB
+  DEVICE_ALT0_MODEL := P.DG A4001N A-000-1A1-AX
+  DEVICE_DTS := agpwi
+  CFE_BOARD_ID := 96328dg2x2
+  CFE_CHIP_ID := 6328
+  FLASH_MB := 8
+  DEVICE_PACKAGES := \
+    $(USB2_PACKAGES) $(B43_PACKAGES)
```

Comment:
This will match into one line.

Reply:
I copied and paste  this too from the A4001N rows and left them as they are for code style uniformity.
Do you still want me to match it in one line?

Reply:
I've just updated this and converted all to a single line.